code/game/objects/effects/landmarks/gamemode/crash_spawns.dm

```diff
@@ -7,93 +7,91 @@
 	if(istype(port, /obj/docking_port/mobile/crashmode))
 		var/obj/docking_port/mobile/crashmode/C = port
 		C.spawnpoints += src
-		if(!C.marine_spawns_by_job[job])
-			C.marine_spawns_by_job[job] = list()
-		C.marine_spawns_by_job[job] += src
+		C.marine_spawns_by_job[job] += list(src)
 
 /obj/effect/landmark/start/marine/crash/captain
```

Comment:
is it worth removing `/marine/` out of all these and maybe calling it `/job/` instead?